main.js

```diff
@@ -432,17 +435,17 @@ if (temp != undefined) {
 //sorting end
 
 // ------------  Build data End  ------------------------
-    dashboard = { "dashboard" : { "pipelines": pipelines , "build": temp, "cloud" : cloud }}; 
-    app.get("/", function(req, res)  {
-        res.json(dashboard);
-    });
+ return dashboard = { "dashboard" : { "pipelines": pipelines , "build": temp, "cloud" : cloud }}; 
+
 }
 app.use(cors());
-main();
 
-setInterval(function() {
-    main();
-},60000 );
+// main();
+
+    app.get("/", function(req, res)  {
+        var branch = req.query.branch;
+        res.json(main(branch));
```

Comment:
@chandankumar4 here it will call main function every  time whenever the api is hit